assets/src/design-system/components/snackbar/snackbarMessage.js

```diff
@@ -118,20 +127,22 @@ const CloseButton = styled(Button)`
 const SnackbarMessage = ({
   actionLabel,
   handleAction = () => {},
-  handleDismiss,
+  handleDismiss = () => {},
```

Comment:
💭 I think most event handlers passed in as props should be named with the `on` prefix whereas the event handlers defined in a component should have the `handle` prefix.

In the future we could rename these to `onActionClick` (or `onAction`) and `onDismiss`. Then we won't need to add `_` to the names of our event handlers. This is def not related to this pr though

Reply:
I agree with this. I'll change it.